recipe/meta.yaml

```diff
@@ -35,11 +35,14 @@ requirements:
     - numpy
     - hdf5
     - cython >=0.29.0  # [py<38]
-    - cython >=0.29.14  # [py>=38]
+    - cython >=0.29.14  # [py==38]
+    - cython >=0.29.15  # [py>=38]
```

Comment:
Is there some reason we need three different versions of cython to build this?

Reply:
The upstream setup file contains three different `cython `https://github.com/h5py/h5py/blob/fb9989a700e60cd1c1dec20e329218d449e039cf/setup.py#L48

Reply:
Since there are upper bounds, I would just set `cython >=0.29.15` as the host requirement.

Reply:
Fixed it as:
`- cython >=0.29.15`